### PR TITLE
adding create_bill_connect python script

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

File: common-bill-ingestion/create_bill_connect.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+'''
+Bill Connect Tool
+---------------------------
+Creates a bill connect and returns the response
+Usage: ./bill_upload_example2.py <refresh_token> <shard> <org_id> <cbi_integration_id> <cbi_bill_identifier> <cbi_name> <cbi_params>
```
Comment (gonzalez) [inline common-bill-ingestion/create_bill_connect.py:5]: Update usage example.

File: common-bill-ingestion/create_bill_connect.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+json_cbi_params = json.loads(cbi_params)
+if type(json_cbi_params) is not type({}):
+  logging.error("cbi_params is not of type dict")
+  sys.exit(1)
+
+logging.info("Using org_id {}".format(org_id))
+
+
+
+token_url = "https://us-"+ shard +".rightscale.com/api/oauth2"
```
Comment (gonzalez) [inline common-bill-ingestion/create_bill_connect.py:56]: https://login.flexera.com/oidc/token - Needs FlexeraOne Compatibility.

File: common-bill-ingestion/create_bill_connect.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+if len(sys.argv) < 8:
+    logging.error('Missing command-line options!!!')
+    print(__doc__)
+    sys.exit(1)
+
+refresh_token, shard, org_id, cbi_integration_id, cbi_bill_identifier, cbi_name, cbi_params = sys.argv[1:]
+logging.info("Using org_id {}, cbi_integration_id {}, cbi_bill_identifier {}, cbi_name {}, cbi_params {}".format(
+             org_id, cbi_integration_id, cbi_bill_identifier, cbi_name, cbi_params))
+
+if not shard == '3':
```
Comment (FabriceScoupeFlexera) [inline common-bill-ingestion/create_bill_connect.py]: Minor: you can also do: `if shard not in ['3', '4']:`

Comment (rshade) [reply common-bill-ingestion/create_bill_connect.py]: @FabriceScoupeFlexera excellent, thank you. I updated that list for F1 also

**Pull request merged**